fotoapparat/src/main/java/io/fotoapparat/hardware/display/Display.kt

```diff
@@ -12,14 +17,14 @@ internal open class Display(context: Context) {
     private val display = context.getDisplay()
 
     /**
-     * Returns the rotation of the screen from its "natural" orientation in degrees.
+     * Returns the orientation of the screen.
      */
-    open fun getRotation(): Int = when (display.rotation) {
-        Surface.ROTATION_90 -> 90
-        Surface.ROTATION_180 -> 180
-        Surface.ROTATION_270 -> 270
-        Surface.ROTATION_0 -> 0
-        else -> 0
+    open fun getOrientation(): Orientation = when (display.rotation) {
+        Surface.ROTATION_0 -> Portrait
+        Surface.ROTATION_90 -> Landscape
+        Surface.ROTATION_180 -> ReversePortrait
+        Surface.ROTATION_270 -> ReverseLandscape
```

Comment:
This looks backwards to me. When I tested this on Samsung GS8, 90 was ReverseLandscape and 270 was Landscape.